springmvc-raml-parser/src/main/java/com/phoenixnap/oss/ramlapisync/data/ApiParameterMetadata.java

```diff
@@ -108,7 +111,8 @@ private String resolveParameterName(String preferredOption, Parameter fallback)
 	 * @param name	The name of this parameter if different in annotation
 	 * @param param Java Parameter representation
 	 */
-	public ApiParameterMetadata(String name, RamlAbstractParam param) {
+	// FIXME Alex - Can we have better JCodeModel handling?
+	public ApiParameterMetadata(String name, RamlAbstractParam param, JCodeModel codeModel) {

```

Comment:
I don't like this part where I have to pass `JCodeModel` around.
Any suggestions @kurtpa ?